core/src/main/java/com/ericsson/bss/cassandra/ecchronos/core/metrics/TableRepairMetricsImpl.java

```diff
@@ -48,11 +53,16 @@ private TableRepairMetricsImpl(Builder builder)
         myNodeMetricHolder = new NodeMetricHolder(myMetricRegistry,
                 Preconditions.checkNotNull(builder.myTableStorageStates, "Table storage states cannot be null"));
 
+        File statisticsDirectory = new File(builder.myStatisticsDirectory);
+        if(!statisticsDirectory.exists() && !statisticsDirectory.mkdirs())
+        {
+            LOG.error("Failed to create statistics directory: {}, csv files will not be generated", builder.myStatisticsDirectory);
```

Comment:
Probably should be a warning since we do not shut down the application with faulty statistics path.

Reply:
Make sense.